src/org/openj9/envInfo/JavaInfo.java

```diff
@@ -164,6 +164,18 @@ public String getJDKVendor() {
         }
     }
 
+    public String getReleaseInfo() {
+        String rt = "";
+        CmdExecutor ce = CmdExecutor.getInstance();
+        String TEST_JDK_HOME = System.getenv("TEST_JDK_HOME");
```

Comment:
Please change the variable name to use the typical standard for naming java variables, lowerCamelCase for these types of local variables.  All caps ([SCREAMING_SNAKE_CASE](https://en.wikipedia.org/wiki/Snake_case)) is reserved for variables denoted as static final keywords.